pkg/controller/kfservice/reconcilers/knative/configuration_reconciler.go

```diff
@@ -68,7 +68,7 @@ func (r *ConfigurationReconciler) reconcileDefault(kfsvc *v1alpha2.KFService) er
 	defaultConfiguration, err := r.configurationBuilder.CreateKnativeConfiguration(
 		constants.DefaultConfigurationName(kfsvc.Name),
 		kfsvc.ObjectMeta,
-		&kfsvc.Spec.Default,
+		&kfsvc.Spec.Default.Predict,
```

Comment:
assume you plan to add other reconcile logic (for explainer, transformer etc.) in separate changes?

Reply:
ye that'd be separate PR.